### ISSUE TITLE
If the "title" required field kept empty the error is not being detected and no error message is displayed: A11y_Visual Studio Code Web Extensions_Github Pull request_Create New Request_Error identification

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode-pull-request-github#7604]
### GIT Hub Tags:
#A11yTCS; #A11ySev2; #Visual Studio Code Web Extensions; #BM_Visual Studio Code Web Extensions_Web_Aug2025; #DesktopWeb; #A11yMAS; #WCAG3.3.1; #Error Identification; #Win11; #ChromiumEdge; #FTP;

### Environment and OS details:
Application name: Visual Studio Code Web Extensions
ChromiumEdge Version: Version 139.0.3405.86 (Official build) (64-bit)
URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github

### Reproduction Steps:

1. Open url https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github environment and login with git hub account.
2. TAB to 'Github Pull request' and press enter.
3. TAB to Create" button and press enter.
4. TAB to "Title" and "Description" field.
5. Observe the issue whether the error is  being detected and error message is displayed or not.

### Actual Behavior:
If the "title" required field kept empty the error is not being detected and no error message is displayed.

### Expected behavior:
If the "title" required field kept empty the error should be detected and error message should be displayed indicating user the way to remove the error.

### User Impact:
When users navigate with a keyboard, they may not be able to identify which field is invalid if there isn’t a visible focus indicator or an error message. Screen reader users will also have trouble if no error appears when a field is left empty, as they won’t understand why their form submission failed or what information is needed.

### Attachment:
https://github.com/user-attachments/assets/1a71b09b-e89c-4020-8634-60d12c4ec080

**Comment by alexr00** [comment]
An error is shown when you try to submit the PR: 

<img width="363" height="194" alt="Image" src="https://github.com/user-attachments/assets/156ff574-32e0-4fa6-a092-992e7a5d0451" />

I think you just didn't try to submit the PR.

**Comment by alexr00** [comment]
Closing as we already have this.

**Comment by kupatkar99** [comment]
#Closed;
